### ISSUE TITLE
Barrier Autostarts in Background on Windows

### ISSUE THREAD

**Issue by saursin** [debauchee/barrier#1756]
### What happened?

I Use Barrier on Linux (KDE Neon) and on Windows 11. 
On Windows, Barrier seems to autostart by itself on login. I don't remember configuring it to autostart. I also checked my list of auto-start apps in windows settings but didn't find barrier. I also tried rebooting several times but It autostarted every time.
I don't think this should be the default behavior of the program.

### Version

v2.4.0

### If applicable, where did you install Barrier from?

2.4.0-release

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

Win-11 Pro 21H2

**Comment by spupuz** [comment]
i suppose there is a barrier service that starts, but i also noticed that the icon in tray in windows is not present.

**Comment by saursin** [comment]
Yes, I think so. I see two processes named `barrierc.exe` (client) and `barrierd.exe` (daemon) in the task manager after boot.
Not sure if that is the intended behavior.

<img width="577" alt="image" src="https://user-images.githubusercontent.com/48968357/186800892-7a7f4062-2d73-4e66-9cc8-4cb8af669b8b.png">

**Comment by rodneyrod** [comment]
The installer creates a service in services.msc, it starts barrierd but not the tray app, also interferes with uninstalls I've found in the past as the service still runs, it cannot be uninstalled and stays installed, and will autostart at startup until it is manually removed.

**Comment by macneilj** [comment]
I also noticed that barrier autostarts in the background on client without having configured it to do so. Not sure if it's intended, but I suppose it's fine for my use case...

Server and client both 2.4.0 on Windows 11.

If it makes any difference, I set both devices to the opposite of what I intended during installation (Server meant to be client and vice versa).
